src/styles/main.scss

```diff
@@ -1,3 +1,239 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
Remove it you already have this prop in * selector

Reply:
Not fixed

Reply:
Not fixed